userspace/libsinsp/threadinfo.h

```diff
@@ -218,6 +218,7 @@ class SINSP_PUBLIC sinsp_threadinfo
 	int64_t m_sid; ///< The session id of the process containing this thread.
 	string m_comm; ///< Command name (e.g. "top")
 	string m_exe; ///< argv[0] (e.g. "sshd: user@pts/4")
+	string m_exepath; ///< full executable path
```

Comment:
You are exposing this field in both `sinsp` and `scap` but it doesn't seem to be used. I think this should either be removed or (better) expose a proper filter field such as `proc.exepath`, so there is at least a consumer for this bit.